src/app/WeatherApp.tsx

```diff
@@ -0,0 +1,62 @@
+import { Col, Row } from "antd";
+import { FaDroplet as HumidityIcon } from "react-icons/fa6";
+import { FaWind } from "react-icons/fa";
+import WeatherIcons from "./weatherIcons";
+import getDate from "./getDate";
+import { useState } from "react";
+
+const WeatherApp: React.FC<{
+  res: {
+    data: {
+      main: { temp: string; humidity: string };
+      name: string;
+      wind: { speed: string };
+      weather: Array<{ main: string }>;
+    };
+  } | null;
+}> = ({ res }) => {
+  if (res === null) {
+    console.log("null");
+  } else {
+    const tempInCalvin = res.data.main.temp;
+    const tempInCelsius = Math.round(Number(tempInCalvin) - 273.15);
+
+    const [changeTime, setChangeTime] = useState(getDate());
+
+    setInterval(() => {
+      setChangeTime(getDate());
+    }, 1000);
+
+    return (
+      <Row className="container">
+        <Col span={5} className="humidity">
+          <div>
+            <div>
+              <HumidityIcon className="icons" /> <br></br> humidity
```

Comment:
Consider some other way of spacing than br tag in cases you are not making just an endline in text. This way has an disadvantage as it is dependent from the font size in the browser.

Maybe: https://ant.design/components/space
or just margin after icon?

See also:
https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br